Makefile

```diff
@@ -367,8 +369,13 @@ run-server: prepackaged-binaries validate-go-version start-docker ## Starts the
 	@echo Running mattermost for development
 
 	mkdir -p $(BUILD_WEBAPP_DIR)/dist/files
+ifeq ($(RUN_SERVER_IN_BACKGROUND),true)
```

Comment:
nit: Can we avoid the duplication of the `run` invocation by conditionally populating a variable with `&` based on this variable instead?

Reply:
let me try